src/features/CourtCaseDetails/Tabs/Panels/Offences/Offence/OffenceDetails.tsx

```diff
@@ -218,13 +216,26 @@ export const OffenceDetails = ({
           <TableRow label="Committed on bail" value={getCommittedOnBail(offence.CommittedOnBail)} />
         </div>
       </Table>
-      <Heading as="h4" size="MEDIUM">
-        {"Hearing result"}
-      </Heading>
-      {offence.Result.map((result, index) => {
-        return <HearingResult result={result} key={index} />
-      })}
-
+      <div className="offence-results-table">
+        <Heading as="h4" size="MEDIUM">
+          {"Hearing result"}
+        </Heading>
+        {offence.Result.map((result, index) => {
+          return (
+            <HearingResult
+              key={index}
+              result={result}
+              exceptions={unresolvedExceptionsOnThisOffence
+                .filter((resultExceptions) => resultExceptions.path[7] === index)
```

Comment:
What's this magic number?

Reply:
The result index position in the exception path. I updated to make it more readable [bcf3b42](https://github.com/ministryofjustice/bichard7-next-ui/pull/467/commits/bcf3b424cb638e6ea0c039c40cc29288c14de1f9)